src/components/_pages/platform-settings/detail/index.tsx

```diff
@@ -23,7 +23,10 @@ export default function PlatformSettingsDetail() {
 
     const getFreshData = useCallback(() => {
         dispatch(utilsActuatorActions.health());
-    }, [dispatch]);
+        if (!platformSettings) {
```

Comment:
I think this condition can be moved to useEffect, like you did with custom attributes.